recipes/common/Makefile

```diff
@@ -0,0 +1,57 @@
+CHROMADBIMAGE ?= quay.io/ai-lab/chromadb:latest
```

Comment:
Which variable is used for the ghcr.io image name?

Reply:
I don't currently do anything with ghcri.io images.

Reply:
The testing framework is consuming the images under tests from ghcr and expected to publish them to quay.io if tests are successful.

Reply:
Changing to 
.PHONY: run
run: 
	podman run -it -p 8501:8501 -e MODEL_SERVICE_ENDPOINT=http://10.88.0.1:8001/v1 ${APPIMAGE}

Reply:
The images should be built locally and pushed when tests pass, but not sure they should change names.

Reply:
None, I changed it to use ${APPIMAGE}